### DISCUSSION TITLE
Can we click a web element by its location (x/y coords)?

### DISCUSSION THREAD

**Discussion by drFrankWhite** [seleniumbase/SeleniumBase#2641] [Q&A] [answered]
How can I use SB to click these coords? {'x': 382, 'y': 698}

**Comment by mdmintz** [comment, marked as answer]
See [examples/test_canvas.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_canvas.py) for clicking on canvas items based on coordinates.

Otherwise, see https://stackoverflow.com/q/6775351/7058266

Or try https://github.com/asweigart/pyautogui

**Comment by TyronMYoung11** [comment]
How to do via `driver = Driver()`?

**Reply by mdmintz** [reply]
You would need to activate CDP Mode, `driver.uc_activate_cdp_mode()`, then then use that API via `driver.cdp`.
Without CDP Mode, the `Driver()` format has much fewer methods than the `SB()` format.
